Update Django OAuth Toolkit to version 2, ref #1944
